MongoDB/src/README.md

```diff
@@ -0,0 +1,248 @@
+
+```python
+    file_id = fs.put(f, content_type='application/pdf', filename='sample_bytes.pdf')
+    # you can provide the filename if you want to upload it with different name
+```
+
+Here, The put() method returned the file_id.
+
+</details>
+<br>
+
+-   Now if you go to the database 'CrioBytes' in your cluster you fill find the collections named 'microbytes.files' & 'microbytes.chunks'
+
+**Task 3: Read the uploaded file using the file_id object**
+
+-   Can you use `get()` & `read()` method provided by `fs` object.
+-   `get()` method returns a file-like-object, and to see the content, we can call `read()` method on this file-like-object.
+-   Will `get()` method require some parameter associated with the file we wanna retrieve? Ofc yes ! To find the file we want, it takes in the file_id.
+
+<details>
+<summary>Hint</summary>
+
+```python
+    print(fs.get(file_id).read())
+    # This will print the content of the file which has id as file_id
+```
+
+</details>
+<br>
+
+**Task 4: Let's insert txt file into MongoDB**
+
+-   Since you have worked with inserting pdf files, this task shall not be difficult one.
+-   You will perform something just like below :
+
+```python
+    f = open('/path_to_your_file/sample_bytes.pdf')
+    fs.put(f, filename="sample_text.txt")
+```
+
+> What if we want to insert the file which is not present on the local system ? Can we insert some text file by inputting the data into that text file on the fly ? (dynamically) !! This might be necessary in some crucial tasks !!! So, lets see how to do so !
+
+-   You might remember, we inserted the simple string "MicroBytes" at the start, and in subsequent task, we inserted text file. Now, the solution is simple, combine/tweak both ?
+
+```python
+    fs.put("Hello, This is a sample string", filename="new.txt")
+```
+
+<br>
+
+**Task 5: Find a file in our database & read its contents**
+
+-   As mentioned Task 2 of Activity 5, the files are stored in db.namespace.files collection. Use this collection to find the file stored.
+-   Use `find_one()` method to find the document inside the collection.
+-   Use `get()` & `read()` to read the contents of file.
+<details>
+    <summary>Hint</summary>
+
+```python
+    file_id = db.microbytes.files.find_one({"filename" : "sample_text.txt"},{"_id":1})
+    print(fs.get(file_id['_id']).read())
+    # Contents of the file `sample_text.txt` will be printed
+```
+
+</details>
+<br>
+
+**Task 6: Delete the file**
+
+-   Now, suppose we want to delete the stored file, then use `delete()` method of `fs` object.
+-   `delete()` method will also require the file_id of the file one wants to delete (Remember ! The `get()` method also needed file_id)
+<details>
+    <summary>Hint</summary>
+
+```python
+    fs.delete(file_id)
+```
+
+</details>
+<br>
+---
+
+Hooray 🎉🎉 We have successfully setup clustes, created database,uploaded, retrieved, deleted files using GridFS file system it through our personal laptop/desktop.
+

```

Comment:
Type- clustes shud be 'clusters'

Reply:
> Make the aforementioned changes. rest looks fine for now :p

Ya I will update the code with required changes and remove typos & structure the content properly